apps/experiments/views/source_material.py

```diff
@@ -33,7 +33,7 @@ class SourceMaterialTableView(SingleTableView):
     template_name = "table/single_table.html"
 
     def get_queryset(self):
-        query_set = SourceMaterial.objects.filter(team=self.request.team)
+        query_set = SourceMaterial.objects.filter(team=self.request.team).exclude(working_version__isnull=False)
```

Comment:
so this only show the source materials associated with a released version? I feel like maybe we should exclude material associated with an archived experiment, but why the working?